scanning/oscap-debug/sysctl-oval-eval-parallel.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/python3
+
+import time
+import subprocess
+import concurrent.futures
+
+from lib import util, results
+
+
+def run(fn):
+    handle = util.subprocess_Popen(['oscap', 'oval', 'eval', fn])
+    try:
+        rc = handle.wait(OSCAP_TIMEOUT)
+        return handle.pid, rc
+    except subprocess.TimeoutExpired:
+        return handle.pid, -1
+
+
+start_time = time.monotonic()
+
+# sysctl ovals only take about 1 second
+OSCAP_TIMEOUT = 5
+
+extra_debuginfos = [
+    'glibc',
+    'openscap-scanner',
+    'xmlsec1',
+    'xmlsec1-openssl',
+    'libtool-ltdl',
+    'openssl-libs',
+]
+util.subprocess_run(['dnf', '-y', 'debuginfo-install', *extra_debuginfos], check=True)
+
+with open('gdb.script', 'w') as f:
+    f.write(util.dedent('''
+        generate-core-file oscap.core
+        set logging file oscap-bt.txt
+        set logging overwrite on
+        set logging redirect on
+        set logging enabled on
+        thread apply all bt
+        set logging enabled off
+    '''))
+
+with util.get_source_content() as content_dir:
+    util.build_content(
+        content_dir,
+        force=True,
+    )
```

Comment:
Why the explicit `force=True`?

Without it (and without extra build options), `build_content()` may save some time by reusing already-built content that's a superset of the (optional) options given to `build_content()`. In this case, I assume you don't care as any content building would have generated the OVAL files, so perhaps:
```suggestion
    util.build_content(content_dir)
```
That is, unless the OVAL files encode absolute paths inserted during content building, in which case `force=True` might be needed.

Reply:
I was just copy-pasting from other test. Most likely I don't need it, any content should do the job.